### PR TITLE
Replace Json with Pickle Serialization

### PR DESCRIPTION
**Pull request by gaurav274**
1. Replace JSON serialization with pickle serialization.
2. Remove unnecessary logging, which was the bottleneck.
3. Reduce the `display.pprint_nest_depth` of ndarray objects in batch

### CODE REVIEW

File: eva/models/storage/batch.py [hunk 1]
```diff
@@ -75,18 +58,14 @@ def columns(self):
...
 
-    def to_json(self):
-        obj = {
-            "frames": self._frames,
-            "batch_size": len(self),
-            "identifier_column": self._identifier_column,
-        }
-        return json.dumps(obj, cls=BatchEncoder)
+    def serialize(self):
+        obj = {"frames": self._frames, "batch_size": len(self)}
```
Comment (xzdandy) [inline eva/models/storage/batch.py:62]: What is the usage for `batch_size`? Can we `return PickleSerializer.serialize(self)`?

File: eva/server/networking_utils.py [hunk 2]
```diff
@@ -79,3 +81,10 @@ def set_socket_io_timeouts(transport, seconds, useconds=0):
         return True
     except OSError:
         return False
+
+
+def serialize_message(message: str):
```
Comment (xzdandy) [inline eva/server/networking_utils.py:86]: Shall we use `typing.Any` for the type hint of `message`?

**Review by xzdandy — APPROVED** [review]
Please check the comments. Overall LGTM

File: eva/models/storage/batch.py (hunk 1 above)
Comment (gaurav274) [reply eva/models/storage/batch.py:62]: For cases where we omit some rows, at least batch_size tells the row count

File: eva/server/networking_utils.py (hunk 2 above)
Comment (gaurav274) [reply eva/server/networking_utils.py:86]: Done

**Pull request merged**